org.eclipse.tm4e.ui/META-INF/MANIFEST.MF

```diff
@@ -15,7 +15,8 @@ Require-Bundle: org.eclipse.tm4e.core,
  org.eclipse.ui.ide;resolution:=optional,
  com.google.gson,
  org.eclipse.e4.ui.css.swt.theme,
- org.eclipse.core.expressions
+ org.eclipse.core.expressions,
+ org.eclipse.ui.genericeditor;bundle-version="1.1.0";resolution:=optional
```

Comment:
Oh yeah, I added the `viewer.getDocument() != null` condition to prevent it from failing with older version of generic editor.